build.sbt

```diff
@@ -796,6 +796,42 @@ lazy val componentsUtils = (project in utils("components-utils")).
     }
   ).dependsOn(componentsApi, commonUtils, testUtils % "test")
 
+//this should be only added in scope test - 'module % "test"' or as dependency to another test module
+lazy val componentsTestUtils = (project in utils("components-test-utils")).
```

Comment:
I wonder if we shouldn't just call it nussknacker-testkit or sth like this (similar to akka). it will be intuitive to look at utils for that?

Reply:
renamed to components-testkit